caddy/caddymain/run.go

```diff
@@ -409,13 +415,61 @@ func initTelemetry() error {
 	return nil
 }
 
+// LoadEnvFromFile loads additional envs if file provided and exists
+//Envs in file should be in KEY=VALUE format
+func LoadEnvFromFile(envFile string) error {
+	if envFile == "" {
+		return nil
+	}
+
+	file, err := os.Open(envFile)
+
+	if err != nil {
+		return err
+	}
+
+	envMap := make(map[string]string)
+
+	scanner := bufio.NewScanner(file)
+	var line string
+
+	for scanner.Scan() {
+		line = scanner.Text()
```

Comment:
I think you should `TrimSpace` for the line instead.

Reply:
Why? i want to sanitize only KEY part, and let value be everything that users want.

Reply:
Extra whitespace is annoying; `export KEY=VAL      ` doesn't set a value of `"VAL      "`, it sets a value of `"VAL"` (obviously without quotes). I think trimming the space is helpful.

Reply:
Oh, ok, i got your point but i need to sanitize key part as well, so i can call TrimSpace on value below? What do you think?

Reply:
Actually, I suggest this with the assumption that a user can encapsulate a value in quotes, since values could also have newlines, rendering trimming of whitespace harmless.

Reply:
Sigh... GitHub didn't show me your latest comment... yes, although on second thought maybe we shouldn't at least for now, unless we honor quotes around the value. But don't worry about it for now.